rero_ils/modules/loans/listener.py

```diff
@@ -22,7 +22,7 @@
 # waive the privileges and immunities granted to it by virtue of its status
 # as an Intergovernmental Organization or submit itself to any jurisdiction.
 
-"""."""
+"""Listener for loans."""
```

Comment:
Can be more verbose such as: Signals connector for Loan.

Reply:
done